### DISCUSSION TITLE
Make -q and -i work together

### DISCUSSION THREAD

**Discussion by 0x7FFFFFFFFFFFFFFF** [microsoft/go-sqlcmd#206] [Ideas]
Right now, both `sqlcmd` and `go-sqlcmd` don't allow you to specify `-q` and `-i` options together. Below is the output of both commands.
```
c:\temp>sqlcmd -S . -E -q "SET PARSEONLY ON" -i test.sql
Sqlcmd: The i and the -Q/-q options are mutually exclusive.
c:\temp>sqlcmdgo -S . -E -q "SET PARSEONLY ON" -i test.sql
sqlcmdgo: error: --input-file and --initial-query can't be used together
c:\temp>
```

If this is possible, it will be very useful. In my case, I have a huge file `test.sql` that is not practical to open in SSMS. What I want is to check its syntax, just like what we normally do by clicking the <kbd>Parse</kbd> toolbar button in SSMS. What SSMS actually does is run the command `SET PARSEONLY ON` before your actually SQL code. If `go-sqlcmd` can allow us to execute a custom command and then an SQL file, it will basically allow us to parse huge SQL files without having to prepend the literal text `SET PARSEONLY ON` at the beginng of the file. This feature might be useful in some other cases, too.  Hope you can consider adding this. Thanks.

**Comment by shueybubbles** [comment]
Thanx for opening the issue!
It's a reasonable request for sure.
In the meantime, you can have multiple `-i` files, so you could put your `SET PARSEONLY ON` in a temp file and include that as the first file in the `-i` list.

**Reply by 0x7FFFFFFFFFFFFFFF** [reply]
Yes, it works! I didn't know you can specify multiple `-i` options. Thanks for sharing the tip!
